main_he.tex

```diff
@@ -0,0 +1,61 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+
+% Language setting
+\usepackage{fontspec} % For font settings
+\usepackage{polyglossia} % Better for handling multiple languages with XeLaTeX or LuaLaTeX
+
+% Set languages
+\setmainlanguage{hebrew} % Set the main language to Hebrew
+\setotherlanguage{english} % Optional: if you have English text as well
+
+% Set main font for Latin script (English)
+\setmainfont{Times New Roman}
+
+% Set font for Hebrew script
+\newfontfamily\hebrewfont{Times New Roman}[
+  Script=Hebrew,
+  Scale=MatchLowercase
+]
+
+% Set font for monospace (if needed)
+\newfontfamily\hebrewfonttt{DejaVu Sans Mono}[
+  Script=Hebrew,
+  Scale=MatchLowercase
+]
+
+% If you need to use monospace font for both scripts
+\setmonofont{DejaVu Sans Mono}
+
+
+
+
```

Comment:
```suggestion
```
Do you really need this? We're not writing any code or math formulas here. Also, please get rid of excessive blank lines.